### DISCUSSION TITLE
Is it possible to add the --debug parameter to argocd's helm commands?

### DISCUSSION THREAD

**Discussion by lozuwa** [argoproj/argo-cd#12003] [General]
Hi, sometimes it's hard to debug what's going on with helm syncs that have a failed state. Is it possible to add the `--debug` parameter? I found this piece of code and it seems like there isn't a way first hand. Thanks. 

[1] https://github.com/argoproj/argo-cd/blob/7165431a8458961a2a19ec5be725904298c19a7d/reposerver/repository/repository.go#L611

**Comment by jaideepr97** [comment]
Hi @lozuwa thanks for raising this
I think you may be right, doesn't hurt to create an issue for it - it should probably be an easy fix
